src/WebJobs.Script.WebHost/FunctionsSyncService.cs

```diff
@@ -42,8 +45,11 @@ internal bool ShouldSyncTriggers
         {
             get
             {
+                var isInValidationMode = FunctionGroups.IsValidationPodGroup(_environment.GetEnvironmentVariable(EnvironmentSettingNames.FunctionsTargetGroup));
```

Comment:
Consider adding an extension method to `IEnvironment` e.g. `IsInValidationMode` so we can remove L#48 and change L#52 to `!_environment.IsInValidationMode()`